src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/eltwise.cpp

```diff
@@ -360,7 +369,9 @@ const std::vector<CPUSpecificParams>& cpuParams_4D_Planar() {
 
 const std::vector<CPUSpecificParams>& cpuParams_4D_PerChannel() {
     static const std::vector<CPUSpecificParams> cpuParams_4D = {
+#if !defined(OV_CPU_WITH_SHL)
         CPUSpecificParams({nhwc, nhwc}, {nhwc}, {}, {}),
+#endif
```

Comment:
If now implement `filterCPUParams`, this function should handle these parameters and this define is not needed. Check please